exercises/reverse-string/README.md

```diff
@@ -0,0 +1,106 @@
+# Reverse String
+
+The introductory exercise to arrays and strings. Just reverse input string.
+
+This is a typical first program for beginning programming in a new language
+or environment after being able to do simple tasks.
+
+The objectives are simple:
+
+- Write a function that reverses the input string.
+- Run the test suite and make sure that it succeeds.
+- Submit your solution and check it at the website.
+
+If everything goes well, you will be ready to fetch your first real exercise.
```

Comment:
I think this line is copied from the hello-world exercise, and it should be removed I think.